src/extension/app-schema/app-schema-test/src/test/java/org/geoserver/test/AbstractAppSchemaMockData.java

```diff
@@ -489,6 +492,11 @@ private static void writeInfoFile(
         }
     }
 
+    private String getLayerName(String typeName) {
```

Comment:
This behavior should not be added here. This is a _specific_ behavior only used in particular test case.

Reply:
Right, done.